### PR TITLE
#322 The "Create" button is active when the user creates a new meeting without a team member

### PR DESCRIPTION
**Pull request by olena-kozak**
https://user-images.githubusercontent.com/98499085/189724488-b903530e-6034-4b8c-b79f-7ef5fc9614ba.mp4

### CODE REVIEW

**Review by YuraRiabov — APPROVED** [review]

File: frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
             type="submit"
             mat-flat-button
             class="new-meeting-header-button"
-            [disabled]="meetingForm.invalid || !meetingForm.touched">
+            [disabled]="meetingForm.invalid || !meetingForm.touched || addedMembers.length === 0">
```
Comment (tatianahutii) [inline frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html]:
```suggestion
            [disabled]="meetingForm.invalid || !meetingForm.touched || !addedMembers.length">
```

**Review by tatianahutii — COMMENTED** [review]
I believe we should not allow users to create meetings without their self

File: frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html (hunk 1 above)
Comment (olena-kozak) [reply frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html]: Added logic to show current user as team member on creating meeting.

**Comment by olena-kozak** [comment]
https://user-images.githubusercontent.com/98499085/189922214-6e344b0d-3da5-4f05-9d1b-5378413a0c8f.mp4

File: frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.ts [hunk 2]
```diff
@@ -155,11 +157,20 @@ export class NewMeetingComponent extends BaseComponent implements OnInit, OnDest
...
+                this.addCurrentTeamMemberToList(resp);
                 this.teamMembers = resp;
-                this.getFilteredOptions();
             });
     }
 
+    private addCurrentTeamMemberToList(meetingMembers: INewMeetingMember[]) {
+        this.userService.userChangedEvent$.subscribe((resp) => {
+            this.currentUser = meetingMembers.find(member => member.id === Number(resp?.id)) as INewMeetingMember;
+
```
Comment (YuraRiabov) [inline frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.ts]: Add null check and remove Number()

**Review by YuraRiabov — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.ts (hunk 2 above)
Comment (olena-kozak) [reply frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.ts]: Fixed it.

**Review by YuraRiabov — APPROVED** [review]

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**